### PR TITLE
Next launch

### PR DESCRIPTION
**Pull request by jakearchibald**

### CODE REVIEW

File: src/_includes/icon-promo.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+.title {
+  font-family: google sans, roboto, sans-serif;
+  color: var(--offblack);
+  margin: 1.4em 0 1em;
+  font-size: 1.8rem;
+  line-height: 1.3;
+}
+
+.blurb {
+  max-width: 25rem;
```
Comment (argyleink) [inline src/_includes/icon-promo.css]:
```suggestion
  max-width: 35ch;
```

pacify scenarios that result in widows

before:
Screen Shot 2019-10-10 at 9 12 38 AM


after:
Screen Shot 2019-10-10 at 9 12 15 AM

**Comment by argyleink** [comment]
I've updated the design comp with latest copy and layout, and also added the intro blurb. ready for review:

<details>

  <summary>Remote Attendee - Desktop v0.0.2</summary>

Desktop - Remote Attendee


</details>

File: src/_includes/subscribe-promo/index.njk [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+{%- set promoCSS = "/_includes/icon-promo.css" -%}
+{% css page, promoCSS %}
+
+<section class="{% className moduleCSS, 'root' %} {% className promoCSS, 'root' %}">
+  <div class="{% className promoCSS, 'icon' %}">
+    <img src="confboxAsset('/assets/youtube-icon.svg')" alt="YouTube">
+  </div>
+  <h1 class="{% className promoCSS, 'title' %}">Can’t attend? No worries.</h1>
+  <p class="{% className promoCSS, 'blurb' %}">The whole event will stream from YouTube and also be available to watch after the event ends 👍</p>
+  <a href="https://www.youtube.com/channel/UCnUYZLuoy1rq1aVMwx4aTzw?sub_confirmation=1" target="_blank" rel="noopener noreferrer" class="{% className promoCSS, 'button' %}">Subscribe</a>
```
Comment (argyleink) [inline src/_includes/subscribe-promo/index.njk]: can you add a `.red-button` class here please so the CTA can have a red shadow?

File: src/_includes/header/style.css [hunk 3]
```diff
@@ -85,20 +85,44 @@
...
   position: absolute;
   left: 50%;
   top: 100%;
   transform: translate(-50%, -50%);
 
+  font-size: 0.7rem;
+  background: var(--white);
+  border-radius: 1000px;
+  padding: 0.9em 0;
+  display: block;
```
[the remark this answers is not in the document]
Comment (jakearchibald) [reply src/_includes/header/style.css]: Good spot!

**Comment by jakearchibald** [comment]
@argyleink have the updated comps been pushed to XD? I'm not seeing them

**Comment by argyleink** [comment]
comps arent updated in XD cloud docs, i still havent resolved the file issue... please use the exported comp provided.

**Comment by jakearchibald** [comment]
No problem

**Comment by jakearchibald** [comment]
Adam said he's happy

**Pull request merged**